README.md

```diff
@@ -205,67 +172,34 @@ Prerequisites:
   [Git for Windows][] includes Git Bash
   and tools which can be included in the global `PATH`.
 
-To build, launch a git shell (e.g. Cmd or PowerShell), run `vcbuild.bat`
-(to build with VS2017 you need to explicitly add a `vs2017` argument),
-which will checkout the GYP code into `build/gyp`, generate `uv.sln`
-as well as the necesery related project files, and start building.
-
-```console
-> vcbuild
-```
-
-Or:
-
-```console
-> vcbuild vs2017
-```
-
-To run the tests:
-
-```console
-> vcbuild test
-```
-
-To see all the options that could passed to `vcbuild`:
-
-```console
-> vcbuild help
-vcbuild.bat [debug/release] [test/bench] [clean] [noprojgen] [nobuild] [vs2017] [x86/x64] [static/shared]
-Examples:
-  vcbuild.bat              : builds debug build
-  vcbuild.bat test         : builds debug build and runs tests
-  vcbuild.bat release bench: builds release build and runs benchmarks
-```
-
+</details>
 
-### Unix
-
-For Debug builds (recommended) run:
+To build with autotools:
 
 ```bash
-$ ./gyp_uv.py -f make
-$ make -C out
+$ sh autogen.sh
+$ ./configure
+$ make
+$ make check
+$ make install
 ```
 
-For Release builds run:
+To build with [CMake][]:
 
 ```bash
-$ ./gyp_uv.py -f make
-$ BUILDTYPE=Release make -C out
-```
-
-Run `./gyp_uv.py -f make -Dtarget_arch=x32` to build [x32][] binaries.
-
-### OS X
+$ cmake -B build -DBUILD_TESTING=ON   # generate project with test
+$ cmake --build build -j 8            # build (8 parallel jobs)
```

Comment:
This is the first time I'm trying to build the project with `cmake` but following the instructions is not working for me. It returns this error: 
```
$ cmake -B build -DBUILD_TESTING=ON
CMake Error: The source directory "/home/sgimeno/software/libuv/build" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
```
The current `cmake` build instructions seem to work correctly though.

Reply:
I'll change it back. `-B` only works with newer versions of cmake.